doc/fertiScan-architecture.md

```diff
@@ -75,69 +92,55 @@ erDiagram
     uuid weight FK
     uuid density FK
     uuid volume FK
-    uuid specification_id FK    
-    uuid first_aid_id FK
-    uuid warranty_id FK
-    uuid instruction_id FK
-    uuid caution_id FK
-    uuid metric_id FK
   }
-
-  specification{
+  sub_label{
```

Comment:
@k-allagbe I've created this entity to represent `instruction, first_aid, specification, warranty`. Like you pointed out, it created alot of code redundancy. With this the additions of new text fields will be more easy and it also create a cleaner code base.

Reply:
I checked. All good. This can be resolved.